packages/tokens/build.js

```diff
@@ -44,42 +64,80 @@ StyleDictionary.registerFormat({
 StyleDictionary.registerFormat({
   name: 'json/dtcg',
   formatter: function ({ dictionary }) {
-    const tokensObject = dictionary.allTokens.reduce(
+    // Returns proper  value for dtcg aliasing
+    const getValue = (value) => {
+      if (value.startsWith('{') && value.includes('.')) {
+        return `${value.split('.')[0]}}`
+      }
+
+      return value
+    }
+
+    // Infers type from attributes. VADS does not consistently populate the type field properly
+    const getType = (attributes) => {
+      const { category, type } = attributes
+
+      if (category.includes('color')) {
+        return 'color'
+      } else if (category === 'units') {
+        return 'dimension'
+      } else if (category === 'font' && type === 'family') {
+        return 'fontFamily'
+      } else if (category === 'font' && type === 'weight') {
+        return 'fontWeight'
+      } else if (category === 'font' && type === 'size') {
+        return 'dimension'
+      }
+
+      return ''
+    }
+
+    // Format tokens for dtcg
+    const tokens = dictionary.allTokens.reduce(
       (previousTokens, token) => ({
         ...previousTokens,
         [token.name]: {
-          $value: token.value,
-          $type: token.path[0], // path[0] is top level token type (e.g. 'color'), should meet: https://tr.designtokens.org/format/#types
+          $value: getValue(token.original.value),
+          $type: getType(token.attributes),
         },
       }),
       {},
     )
 
-    return JSON.stringify(tokensObject, undefined, 2) + `\n`
+    return JSON.stringify(sortTokensByKey(tokens), undefined, 2) + `\n`
   },
 })
 
-/** Registering a transform that strips out category from token name */
-StyleDictionary.registerTransform({
-  name: 'name/color/clean-up',
-  type: 'name',
-  transformer: (token) => {
-    return token.name.replace('SystemColor', '')
-  },
-})
+/**
+ * Transform Groups
+ */
 
 /** Registering transform group to massage output as desired for figma */
 StyleDictionary.registerTransformGroup({
   name: 'rn',
-  transforms: ['name/ti/camel', 'name/color/clean-up', 'color/hex'],
+  transforms: ['name/cti/camel', 'color/hex'],
 })
 
 /** Registering transform group to massage output as desired for figma */
 StyleDictionary.registerTransformGroup({
   name: 'figma',
-  transforms: ['name/ti/camel', 'name/color/clean-up', 'color/hex'],
+  transforms: ['name/cti/kebab', 'color/hex'],
```

Comment:
Did we change this to kebab for consistency with VADS/USWDS? If so, perhaps we should also change the tokens themselves to kebab? 

I think we should maintain consistency as otherwise it adds unnecessary cognitive load for designers/engineers to be communicating token names slightly differently.

Reply:
As a follow-up thought, not sure if that'd be annoying in the code then requiring all token references to wrap kebab-case names in a string or something. To which, if so, I'd push for Figma using camelCase as well and chalking it up as a meaningful web vs mobile differentiation as HTML more naturally using kebab-case while JS/TS doesn't like kebab-case naming.

Reply:
I think as long as the names are the same (e.g. `UswdsSystemColorBlueVivid60` in JS and `uswds-system-color-blue-vivid-60` for figma) I'm fine with the difference in casing and I don't think there'll be much cognitive load in communicating. Previously I was filtering out `vads` and `SystemColor` so undoing that keeps the names identical to the original but with different casing. 

@jessicawoodin expressed an interest in keeping the token names the same as VADS for figma because at some point, we're going to have to move over to VA's Figma and we'll be directly referencing their tokens which will be in kebab case, and kebab case just doesn't make sense for JS. VADS will likely want to continue to use kebab case since they're mainly working in CSS.

Reply:
Not a fan of the inconsistency, but fair play to this part overruling:
> keeping the token names the same as VADS for figma because at some point, we're going to have to move over to VA's Figma